docs/examples.md

```diff
@@ -0,0 +1,19 @@
+# Sister projects, examples & articles
+
+## Sister projects
+
+* [qemu-user-static (qus) and docker](https://github.com/dbhi/qus)
+
+## Examples and articles [DEPRECATED]?
+
+* Scaleway's build system:
+  * [scaleway/image-tools](https://github.com/scaleway/image-tools)
```

Comment:
From what I know, they changed their way of managing multiarch a little bit but they are still doing multiarch images

Reply:
You want me to change this part of document? Or just you shared what you know, right?